src/test/function_test/test_restore.cpp

```diff
@@ -256,19 +267,20 @@ class restore_test : public testing::Test
 
     bool find_second_backup_timestamp()
     {
-        std::string policy_dir = dsn::utils::filesystem::path_combine(cluster_name, policy_name);
         std::vector<std::string> dirs;
         ::dsn::utils::filesystem::get_subdirectories(policy_dir, dirs, false);
         return (dirs.size() >= 2);
     }
 
 public:
     pegasus_client *pg_client;
+    pegasus_client *new_pg_client;
```

Comment:
In my view, `new_pg_client` is used to operate restore table?

Reply:
Yes, you are right.